src/lib/testUtils.js

```diff
@@ -12,10 +12,12 @@ module.exports = {
 
   createTestUserOpts: (number = 0) => {
     const randomNum = _.random(10000) + number;
+    const name = `name${randomNum}`;
```

Comment:
I know this is just for testing, but couldn't we have kept the name just the name instead of concatenating the `randomNum`?

Reply:
Theres a unique constraint on the name field